HW2/P3/P3.txt

```diff
@@ -0,0 +1,24 @@
+Using AVX instructions results in a significant speed up. For example, using one thread, execution time without AVX is 4.5 sec, and with AVX is 0.6 sec(more than 7 times speed up).
+
+	--Results chunksize=1, schedule=static---
+num_threads= 1
+1074.656613 Million Complex FMAs in 4.56919312477 seconds, 235.19614594 million Complex FMAs / second
+
+num_threads= 2
+1074.656613 Million Complex FMAs in 2.22668504715 seconds, 482.626231481 million Complex FMAs / second
+
+num_threads= 4
+1074.656613 Million Complex FMAs in 1.14343690872 seconds, 939.847756184 million Complex FMAs / second
+
+num_threads= 8
+1074.656613 Million Complex FMAs in 0.836094856262 seconds, 1285.32857839 million Complex FMAs / second
+
+---8 way instruction---
+num_threads=1
+Mandelbrot AVX 1074.9472 Million Complex FMAs in 0.635610103607 seconds, 1691.20533783 million Complex FMAs / second
+num_threads=2
+Mandelbrot AVX 1074.9472 Million Complex FMAs in 0.333503007889 seconds, 3223.20091445 million Complex FMAs / second
+num_threads=4
+Mandelbrot AVX 1074.9472 Million Complex FMAs in 0.172398090363 seconds, 6235.26164205 million Complex FMAs / second
+num_threads=8
+Mandelbrot AVX 1074.9472 Million Complex FMAs in 0.115933895111 seconds, 9272.07007899 million Complex FMAs / second
```

Comment:
Agreed! A plot would again be nice though. You must be using a crazy-powerful computer...10Gflops! Wow!